.github/workflows/test-and-build.yaml

```diff
@@ -99,5 +99,5 @@ jobs:
               owner: 'statisticsnorway',
               repo: '${{ secrets.INTEGRATION_TEST_REPO }}',
               workflow_id: '${{ secrets.INTEGRATION_TEST_WORKFLOW }}.yaml',
-              ref: 'master'
```

Comment:
oof I've missed this ref, this needs to be fixed on all datastore relevant repositories 👀  good catch :100: